.eslintrc.json

```diff
@@ -10,6 +10,7 @@
     "mocha": true
   },
   "rules": {
-    "prettier/prettier": ["error", { "singleQuote": true, "semi": false }]
+    "prettier/prettier": ["error", { "singleQuote": true, "semi": false }],
+    "no-unused-vars": ["error", { "argsIgnorePattern": "^__" }]
```

Comment:
@davesag can't tell if I'm doing something wrong.  I run "npm run lint:sol" and it doesn't seem to ignore this problem.  Same with truffle compile.

Reply:
In the StakeBank contract there is a field in stakeFor and unstake called __data which is unused and is supposed to be ignored.